ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json

```diff
@@ -1688,7 +1688,7 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "manageDocumentsCourtBundleNoHearingsLabel",
```

Comment:
added the original field to AuthorisationCaseEvent/CareSupervision/DEPRECATED.json